app/styles/_flip-card.scss

```diff
@@ -0,0 +1,76 @@
+.flip-card {
+  border: none;
+  position: relative;
+}
+
+.flip-card-cover,
+.flip-card-figure {
+  animation-duration: 0s;
+  animation-fill-mode: forwards;
+  backface-visibility: hidden;
```

Comment:
For Safari to work correctly we need to set `-webkit-backface-visibility: hidden;`. The rest of the styles work great on Safari.